docs/finder-content-item.md

```diff
@@ -0,0 +1,61 @@
+# The Finder Content Item Format
+
+A Finder Content Item is a specialisation of the [Content Item](https://github.com/alphagov/content-store/blob/master/doc/content_item_fields.md). This guide explains what goes in the details hash of the ContentItem and why.
+
+The Finder Content Item is used by Finder Frontend to render the Finder page. Most of what it uses goes in the `details` hash.
+
+# The `details` hash
+
+## `beta`
+
+A boolean. Required.
+
+A flag used to decide if the Beta banner should be rendered.
+
+## `beta_message`
+
+A string. Required.
```

Comment:
If `beta` is false is this still required? Even if `beta` is true, isn't there a default beta message that can be used if this isn't provided?